src/concepts/components/trapevents.md

```diff
@@ -0,0 +1,311 @@
+---
+description: Component could mutate internal state or re-render based on events emitted by html or yew components
+---
+
+# Trap events and mutate state
```

Comment:
What is a 'trap event'?

Reply:
I used this word in the concept to "handle", "Listen", "register" to events generated by HTML component
I will use => Listen